src/Azure.Functions.Cli/Helpers/AzureHelper.cs

```diff
@@ -423,9 +423,41 @@ public static async Task<Site> LoadSiteObjectAsync(Site site, string accessToken
             site.Kind = armSite.kind;
             site.Sku = armSite.properties.sku;
             site.SiteName = armSite.name;
+            site.FunctionAppConfig = armSite.properties.functionAppConfig;
             return site;
         }
 
+        public static async Task<bool> UpdateFlexRuntime(Site site, string runtimeName, string runtimeValue, string accessToken, string managementURL)
+        {
+            var url = new Uri($"{managementURL}{site.SiteId}?api-version={ArmUriTemplates.WebsitesApiVersion}");
+            var functionAppJson = await ArmHttpAsyncForFlex(HttpMethod.Get, url, accessToken);
+            dynamic functionApp = JsonConvert.DeserializeObject(functionAppJson);
+            var runtimeConfig = functionApp?.properties?.functionAppConfig?.runtime;
+
+            if (runtimeConfig == null)
+            {
+                return false;
+            }
+
+            runtimeConfig.name = runtimeName;
+            runtimeConfig.version = runtimeValue;
+
+            string functionAppString = JsonConvert.SerializeObject(functionApp);
+
+            url = new Uri($"{managementURL}{site.SiteId}?api-version={ArmUriTemplates.FlexApiVersion}");
+            await ArmHttpAsyncForFlex(new HttpMethod("PATCH"), url, accessToken, functionApp);
```

Comment:
AFAIK, we currently can't do a PATCH request to update attributes, we'll need to do a PUT.

Reply:
Thanks @kamperiadis!  Can you please share the recommended API. I got it from my local AZ CLI copy which was using Patch. Interestingly, it is actually working in beta :).

Reply:
I just confirmed with David today - we did not do any implementation changes for PATCH, which might be missing things like blocking unsupported configs in the Site object. I would not suggest using PATCH. Instead, we should be doing a GET call to the function app and then do a PUT request with that same function app object but with the updated values. 

Not sure what Az CLI copy you have but for flex specifically, the Az CLI makes PUT calls when updating the runtime.

Reply:
Changed it to us `PUT` instead of `PATCH`.